hf/run_dpo_no_trainer.py

```diff
@@ -105,34 +109,37 @@ def eval_fn(model, ref_model, eval_device_loader, config, step, tracker):
     )
 
 
-@hydra.main(version_base=None, config_path="config", config_name="config")
+def hydra_decorator(config_path, config_name):
```

Comment:
what does this part mean?

Reply:
hydra config is powerful and I do expect to use it in gpu as well.

It is also used by nvidia's nemo megatron
https://github.com/NVIDIA/NeMo-Aligner/blob/main/examples/nlp/gpt/train_gpt_dpo.py#L44

Reply:
Yep, as you pointed out, the GPU path uses this `hydra_runner` from `nemo.core.config`, so that's why I refactor it into a conditional decorator

Reply:
Thank you!